src/scripts/plugin.js

```diff
@@ -14,6 +14,23 @@ const languageDictionary = {
   SageMath: ['SageMath'],
 };
 
+const dataLanguageDictionary = {
+  python3: ['Python 3'],
+  julia: ['Julia'],
+  R: ['R'],
+  octave: ['Octave'],
+  SageMath: ['SageMath'],
+};
+
+const getLanguage = (editor) => {
+  const blockList = editor.document.getElementsByTag('pre');
```

Comment:
What if there is a normal `pre` block that the author inserted before any block? Shouldn't we just change the ones that have `data-executable`?

Reply:
Hopefully addressed in 2bde441